packages/core/strapi/src/load/glob.ts

```diff
@@ -1,15 +1,15 @@
-import glob, { IOptions } from 'glob';
+import { glob } from 'glob';
 
 /**
  * Promise based glob
  */
-function promiseGlob(...args: [string, IOptions]): Promise<string[]> {
-  return new Promise((resolve, reject) => {
-    glob(...args, (err, files) => {
-      if (err) return reject(err);
-      resolve(files);
-    });
-  });
+async function promiseGlob(...args: [string, any]): Promise<string[]> {
```

Comment:
This function has become unnecessary as glob is now async, you can remove this file and update `packages/core/strapi/src/load/load-files.ts` to use glob directly I think

Reply:
Makes sense, on it

Reply:
@alexandrebodin I've updated